langkit/examples/Intro_to_Langkit.ipynb

```diff
@@ -58,7 +58,7 @@
     "from langkit import llm_metrics # alternatively use 'light_metrics'\n",
     "import whylogs as why\n",
     "\n",
-    "why.init()\n",
+    "why.init(upload_on_log=True)\n",
```

Comment:
need to update whylogs dependency range to >=1.5.0, and make this change after we release

Reply:
The whylogs dependency is updated, but we'll need to bump the version of langkit installed in the notebooks.

Reply:
Right, should we make the pyproject.toml dependency change first, release and then update the notebooks so the install points to a publicly available version? I'm concerned the notebooks will be broken after we merge this until we release.